### ISSUE TITLE
Waiting from proxmark to appear

### ISSUE THREAD

**Issue by RJswap** [Proxmark/proxmark3#914]
Hi !

I am a new user of proxmark3. I therefore followed every steps of the wiki tutorial to flash my proxmark. I tried the procedure on both kali and ubuntu but when I flash I got the message"waiting from proxmark to appear on /dev/ttyACM0...". (The dmesg command worked as expected and the proxmark is seen on ttyACM0). First I was running on a windows subsystem but then i tried on a linux (ubuntu) laptop but the problem remains. I tried some solutions found on internet such as using sudo but none of them solved the issue.
Do you know what could be wrong with my attempt ?

Thanks for your help

**Comment by pwpiwi** [comment]
Did you read about modem_manager?

**Comment by RJswap** [comment]
Thanks for your reply,
Yes I tried the solution suggested on proxmark.org about modem_manager

**Comment by pwpiwi** [comment]
Could you solve your problem? If yes, would you mind to tell the community what went wrong in first place?

**Comment by RJswap** [comment]
Hi,
No, I still could not solve my problem. 
at first I used an ubuntu virtual machine on windows (I thought there were problems with authorization of access to usb devices); then i installed a whole ubuntu os on a new machine but the problem remains the same

**Comment by pwpiwi** [comment]
Please send us some screenshots or console cut&paste from dmesg, proxmark3, and flasher commands. Both your input and the respective output.

**Comment by pwpiwi** [comment]
Are you still there?

**Comment by Aquarh** [comment]
Yes but with confinement I no longer have access to my proxmark so I am not able to provide screenshots. I haven't given up x)

**Comment by seanjwaters** [comment]
I'm currently having the same issues. I'm following [this guide](https://hackerwarehouse.tv/product-knowledgebase/proxmark/proxmark3-rdv4-setup-in-kali-linux/). Everything seems to be normal until I get to the _Flashing the Proxmark3 RDV4_ section. Screenshots linked below.

[dmesg](https://i.imgur.com/Cfh0thF.png)
![flashing-waiting](https://user-images.githubusercontent.com/72681216/112095147-f436ef80-8b69-11eb-87cd-9c38b0f53bf7.png)
[flashing-error](https://i.imgur.com/6YvP5kJ.png)

**Comment by iceman1001** [comment]
I suggest following the instructions on RRG/Iceman repository if you want to install it on your RDV4,  for the most up-to-date guides.
